### ISSUE TITLE
Should all args in a command be converted into Lua's syntax?

### ISSUE THREAD

**Issue by delphinus** [nvim-telescope/telescope.nvim#437]
**Is your feature request related to a problem? Please describe.**

When I call `:Telescope` with builtin config values in args, [convert_user_opts][] parse them and translate vimL values to Lua's ones.

[convert_user_opts]: https://github.com/nvim-telescope/telescope.nvim/blob/00e898a1f919a2330c3b0fdf21429627cbaf2175/lua/telescope/command.lua#L9

```lua
:Telescope find_files color_devicons=false

=> opts = {color_devicons = false}
```

But this does not parse other opts defined in each subcommand. I found this in developing #430.

```lua
:lua require'telescope.builtin'.help_tags{fallback = false}

=> opts = {fallback = false}

:Telescope help_tags fallback=false

=> opts = {fallback = 'false'}
```

**Describe the solution you'd like**

I think such opts should be also converted into Lua values in the same way. How do you think?

**Describe alternatives you've considered**

**Additional context**

**Comment by Conni2461** [comment]
I ran into the same issue and wanted to ask @glepnir if this was possible :)

**Comment by glepnir** [comment]
should be possible. I will improve it later and test.

**Comment by glepnir** [comment]
@delphinus  now in my local this can work, other request?
```
:Telescope help_tags fallback=false
``

**Comment by delphinus** [comment]
Yeah, that's it ;)

**Comment by glepnir** [comment]
closed by #439
